web-components/src/components/menu-overlay/MenuOverlay.ts

```diff
@@ -78,6 +79,7 @@ export namespace MenuOverlay {
     @property({ type: Boolean }) disabled = false;
     @property({ type: String }) placement: MenuOverlay.Placement = "bottom";
     @property({ type: Boolean, attribute: "allow-hover-toggle" }) allowHoverToggle = false;
+    @property({ type: String, attribute: "custom-role" }) customRole = "menu";
```

Comment:
Can you create type as Role?
Refer web-components/src/components/button/Button.ts 

Also, since its a new property, update UTs and sandbox examples